WebKitFuzz/build_webkitfuzz.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+echo "[*] Building webkitfuzz."
+
+export CC=/usr/bin/clang
+export CXX=/usr/bin/clang++
+
+# -g flag for debugging symbols
+# -w to skip printing warnings
+# -Wfatal-error to immediately stop build with an error is detected
+export CFLAGS="-fsanitize=address -g -w -Wfatal-error"
+export CXXFLAGS="-fsanitize=address -g -w -Wfatal-error"
+export LDFLAGS="-fsanitize=address -g"
+export ASAN_OPTIONS="detect_leaks=0"
+
+mkdir build
+cd build
+
+echo "[*] webkitfuzz: running cmake"
+
+# Explanation of cmake flags:
+# -DCMAKE_BULD_TYPE=Release -DPORT=GTK -- Build release build of WebKit GTK port
+#
+# -G "Unix Makefiles". Change to -G "Ninja" if you want to build with ninja
+#
+# -DCMAKE_INSTALL_PREFIX=. -DCMAKE_INSTALL_LIBEXECDIR=libexec/
+# -DLIB_INSTALL_DIR=lib/ -DCMAKE_SKIP_RPATCH=ON - Required to have all the build
+# files and libs end up in your build/ directory so webkitfuzz uses those files
+# instead of the default on your machine
+#
+# -DENABLE_SANITIZERS=address - Build with ASAN
+# -DENABLE_MINIBROWSER=ON - webkitfuzz uses minibrowser
+#
+# The rest turn off dependencies not needed for most fuzzing cases
+cmake -DCMAKE_BUILD_TYPE=Release -DPORT=GTK -G "Unix Makefiles" \
+  -DCMAKE_INSTALL_PREFIX=. -DCMAKE_SKIP_RPATH=ON -DLIB_INSTALL_DIR=./lib \
+  -DCMAKE_INSTALL_LIBEXECDIR=./libexec \
+  -DENABLE_SANITIZERS=address \
```

Comment:
IIRC you planned on using system malloc, `-DUSE_SYSTEM_MALLOC=ON`, and it's also mentioned in the documentation, but isn't included here.